config/crds/kr6.0_serving_v1alpha1_300-clusteringress.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 The Knative Authors
```

Comment:
This file is not required for the tests.

Reply:
removed